Networking/Networking/Model/Product/Product.swift

```diff
@@ -109,6 +109,10 @@ public struct Product: Codable, GeneratedCopiable, Equatable {
         return ProductTaxStatus(rawValue: taxStatusKey)
     }
 
+    public var attributesForVariations: [ProductAttribute] {
```

Comment:
nit: It would be nice to add a comment and explain why this is needed to future readers.